src/OrchardCore.Modules/OrchardCore.Admin/Views/AdminSettings.Edit.cshtml

```diff
@@ -1,9 +1,144 @@
 @model OrchardCore.Admin.ViewModels.AdminSettingsViewModel
 
-<fieldset class="form-group" asp-validation-class-for="Zones">
+<script asp-src="~/OrchardCore.Media/Scripts/media.js" asp-name="media" at="Foot" depends-on="admin, vuejs, sortable, vuedraggable"></script>
+<style asp-src="~/OrchardCore.Media/Styles/media.min.css" debug-src="~/OrchardCore.Media/Styles/media.css"></style>
+
+<script at="Foot">
+    initializeMediaApplication(false,  '@Url.Action("MediaApplication", "Admin", new { area = "OrchardCore.Media" })');
+    initializeMediaField(
+        document.getElementById('@Html.IdFor(m => m.BrandImage)'),
+        document.getElementById('@Html.IdFor(m => m.BrandImage)-ModalBody'),
+        '@Url.Action("GetMediaItem", "Admin", new { area = "OrchardCore.Media" })',
```

Comment:
This method is what loads the asset(s) on the media picker

Reply:
I'm debugging to see what you mean.

Reply:
I tried this but it doesn't work:

`<input asp-for="BrandImage" type="hidden" class="paths" :value="paths" data-init="[{ 'Paths': '@Model.BrandImage' }]" />`

Reply:
Ok there is a bug in resolving the assets in the media library.

Reply:
I have the same issue here. I'm trying to find why `:value="paths"` doesn't work. If we test with a Field it makes a GET call to `GetMediaItem?path=yourfile.jpg` but it never calls it on this admin settings view. We pass now the proper JSON to the component in the `data-init` html attribute so if it's not working there's something happening in the Vue.js component.